src/entities/data-export/data-export.ts

```diff
@@ -131,6 +131,25 @@ export class DataExport extends UniverseEntity<DataExportPayload, DataExportRawP
       throw this.handleError(new DataExportInitializationError(undefined, { error: err }))
     }
   }
+
+  public async exportCsv (payload: { filters: Array<{ id: string, values: string[] | string[][] }> } = { filters: [] }): Promise<any | undefined> {
```

Comment:
would be nice to have typing of the promise at that point but maybe not that important 🙏